test-e2e/cip/push-golden.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# About: This script automatest the process of pushing golden iamges for e2e tests.
+# Images are loaded from local archives and pushed to the designated staging repo.
+# The script is triggered within e2e.go during test setup.
+#
+# Usage:
+#   ./push-golden.sh repo-root
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [[ $# -ne 1 ]]
+then
+    printf "Error: Invalid number of arguments!\n\tUsage: publish-golden.sh repo-root\n"
+    exit 1
+fi
+
+repoRoot=$1
```

Comment:
For your script the invocation would be

```
$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)
```

(notice the extra `/..`) because the script is 2 directory levels below the root (unlike test-go.sh, which is only 1 directory level below the root). Does this invocation also not work?

Reply:
Yes! This works when invoking push-golden.sh from any directory.
Thank you